packages/venia-concept/src/components/MiniCart/product.css

```diff
@@ -86,13 +64,13 @@
     display: inline-flex;
 }
 
-.modal {
+.mask {
     position: absolute;
     left: -24px;
     right: -24px;
     top: -7px;
     bottom: -7px;
-    background-color: rgb(var(--venia-teal));
-    opacity: 0.1;
+    background-color: rgb(var(--venia-grey));
+    opacity: 0.5;
```

Comment:
@soumya-ashok The opacity here could be increased if we want. I leave that to a UX decision. You can see in the GIF I posted earlier how it looks, or I can show you if you come by my desk :)